screwdriver.yaml

```diff
@@ -0,0 +1,69 @@
+shared:
+  environment:
+    DOCKER_REPO: screwdrivercd/artifacts-unzip-worker
+
+jobs:
+  main:
+    image: node:16
+    environment:
+      SD_SONAR_OPTS: "-Dsonar.sources=lib,plugins,helpers -Dsonar.tests=test -Dsonar.javascript.lcov.reportPaths=artifacts/coverage/lcov.info -Dsonar.testExecutionReportPaths=artifacts/report/test.xml"
+    requires: [~pr, ~commit]
+    steps:
+      - install: npm install
+      - test: npm test
+
+  # Publish the package to GitHub and build Docker image
+  publish:
+    image: node:16
+    requires: [main]
+    steps:
+      - setup-ci: git clone https://github.com/screwdriver-cd/toolbox.git ci
+      - install-ci: npm install npm-auto-version
+      - publish-npm-and-git-tag: ./ci/publish.sh
+    secrets:
+      # Publishing to NPM
+      - NPM_TOKEN
+      # Pushing tags to Git
+      - GIT_KEY
+
+  docker-publish:
+    requires: publish
+    template: sd/dind@latest
+
+  # Deploy to beta environment and run tests
```

Comment:
I comment out deploy jobs because it does not work until the all sd-zip-artifacts function has come. It just takes resources.